index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { Linking } from 'react-native'
 
-export default (Component, handler) => class LinkingAwareComponent extends React.Component {
+export default (Component, handler, options = {}) => class LinkingAwareComponent extends React.Component {
```

Comment:
I'd avoid introducing the term `options` for such a minute configuration. How about simple destructuring?